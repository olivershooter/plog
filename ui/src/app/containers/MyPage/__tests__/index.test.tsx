import React from 'react';
import { render, screen } from '@testing-library/react';
import { describe, it } from 'vitest';
import { MyPage } from '..';

describe('<MyPage />', () => {
  it('renders and matches the snapshot', () => {
    const { asFragment } = render(<MyPage />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('should contain the title MyPage', () => {
    render(<MyPage />);

    expect(screen.getByText('Contact')).toBeInTheDocument();
  });
});
