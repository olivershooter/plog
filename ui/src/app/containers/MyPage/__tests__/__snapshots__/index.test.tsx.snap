// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Contact /> > renders and matches the snapshot 1`] = `
<DocumentFragment>
  <h1>
    Contact
  </h1>
</DocumentFragment>
`;
